README.md

```diff
@@ -16,15 +17,22 @@ This a mobile version skeleton of portfolio.
 - The textarea/message box accepts a maximum of 500 characters.
 - Form is having an interactive button.
 
+### (Update) Use JavaScript to implement the following functions:
+- Click on the hamburger menu icon to show menu options.
+- Click on the X (close button) to close menu. 
+- Click on the menu option to show that particular option. 
```

Comment:
- [OPTIONAL] Your README file should contain the description, how to use it, acknowledgments, contact information, etc. This piece of information looks misplaced in the README and I would remove it.